### DISCUSSION TITLE
硕士论文封面问题

### DISCUSSION THREAD

**Discussion by eutopia-61** [note286/xduts#60] [Q&A] [answered]
封面的信息怎么设置对齐啊，我看以往封面名字都是对齐
![image](https://user-images.githubusercontent.com/67130065/223084004-7c7d3e4d-0c07-43ee-9e9b-51cc687478c4.png)

**Comment by note286** [comment, marked as answer]
我精心设计的样式，在你看来竟然是没对齐！(灬ꈍ ꈍ灬)

![image](https://user-images.githubusercontent.com/16207292/223084713-6d7bcc84-91d8-4837-9cb9-be9616281a8e.png)

**Reply by eutopia-61** [reply]
不好意思，就我看那个word模板里。名字都是左对齐
![image](https://user-images.githubusercontent.com/67130065/223085758-5d593662-4901-4b36-b70c-588d92da533b.png)

**Reply by note286** [reply]
美观即可，没有强制要求。

你觉得哪个好看。作为作者，我对自己的设计很满意。

**Reply by eutopia-61** [reply]
好的好的，感谢

**Reply by note286** [reply]
而且Microsoft Word也不是真的对齐，空格空出来的吧。

![image](https://user-images.githubusercontent.com/16207292/223086468-08f472d0-6f11-4889-bf08-71e345953cdc.png)

**Reply by eutopia-61** [reply]
他每个名字都用了一个控件，反正是这么摆着的，我没敲空格

**Reply by note286** [reply]
我说Microsoft Word模板，不是说你主动敲空格。我指的是`作者姓名`和`张三`之间这一段是空格。

**Reply by eutopia-61** [reply]
是的

**Reply by note286** [reply]
Microsoft Word模板是很草率的，都不能称之为“模板”，基本没有帮助同学们设置好什么，写论文的时候还要自己处理好多，很多细节都没有注意，敷衍了事。

另外，star也不给一个。

**Reply by eutopia-61** [reply]
行，我再好好看看

**Reply by sikouhjw** [reply]
> Microsoft Word模板是很草率的，都不能称之为“模板”，基本没有帮助同学们设置好什么，写论文的时候还要自己处理好多，很多细节都没有注意，敷衍了事。

不得不推荐我的~高质量~ Word 模板了 https://github.com/sikouhjw/gdutthesis-Word

**Reply by note286** [reply]
我Microsoft Word的相关能力，只有安装和激活。

**Reply by Seven-zhanghf** [reply]
您好，我是专硕，应该是和题主的图片一样。但是用模板导出如图。阅读了文档不知道用哪个参数配置。
<img width="498" height="183" alt="123456" src="https://github.com/user-attachments/assets/cdf88670-262b-49b7-95c0-26fb94dc7d7a" />

**Reply by note286** [reply]
@Seven-zhanghf  看手册，里面都有
```latex
\xdusetup{info/degree-type=专业}
```

**Reply by Seven-zhanghf** [reply]
> @Seven-zhanghf 看手册，里面都有
> 
> ```
> \xdusetup{info/degree-type=专业}
> ```

好的，是我的问题，信息都写了，写错这个了。谢谢
